slices/stunnel4.yaml

```diff
@@ -0,0 +1,30 @@
+package: stunnel4
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libc6_config
+      - libssl3_libs
+      - openssl_config
+      - libgssapi-krb5-2_libs
+      - libtirpc3_libs
```

Comment:
Also, some of the listed essentials do not seem to be either direct or indirect dependencies of `stunnel4`, so I wonder why they are listed here. Can you please explain the dependency tree? If the idea is that some of these essentials are "useful" but not really a dependency, then they shouldn't be here, and instead, they should be explicitly installed when doing `chisel cut`.

Here's the control file for `stunnel4`: 
```
Package: stunnel4
Version: 3:5.63-1build1
Priority: optional
Section: universe/net
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: Peter Pentchev <roam@debian.org>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 569 kB
Provides: stunnel
Pre-Depends: adduser
Depends: libc6 (>= 2.34), libssl3 (>= 3.0.0~~alpha1), libsystemd0, libwrap0 (>= 7.6-4~), init-system-helpers (>= 1.51), systemd | systemd-standalone-sysusers | systemd-sysusers, perl:any, lsb-base, netbase, openssl
Suggests: logcheck-database
Homepage: https://www.stunnel.org/
Download-Size: 215 kB
APT-Sources: http://ch.archive.ubuntu.com/ubuntu jammy/universe amd64 Packages
...
```

Reply:
Here is the alternate logic for deciding which files are needed by `stunnel4`:

```
$ ldd /usr/bin/stunnel4                    
	linux-vdso.so.1 (0x00007ffeaa9d6000)
	libssl.so.3 => /lib/x86_64-linux-gnu/libssl.so.3 (0x00007f899f544000)
	libcrypto.so.3 => /lib/x86_64-linux-gnu/libcrypto.so.3 (0x00007f899f102000)
	libsystemd.so.0 => /lib/x86_64-linux-gnu/libsystemd.so.0 (0x00007f899f03b000)
	libwrap.so.0 => /lib/x86_64-linux-gnu/libwrap.so.0 (0x00007f899f02f000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f899ee07000)
	liblzma.so.5 => /lib/x86_64-linux-gnu/liblzma.so.5 (0x00007f899edda000)
	libzstd.so.1 => /lib/x86_64-linux-gnu/libzstd.so.1 (0x00007f899ed0b000)
	liblz4.so.1 => /lib/x86_64-linux-gnu/liblz4.so.1 (0x00007f899eceb000)
	libcap.so.2 => /lib/x86_64-linux-gnu/libcap.so.2 (0x00007f899ece0000)
	libgcrypt.so.20 => /lib/x86_64-linux-gnu/libgcrypt.so.20 (0x00007f899eba2000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f899f65b000)
	libnsl.so.2 => /lib/x86_64-linux-gnu/libnsl.so.2 (0x00007f899eb88000)
	libgpg-error.so.0 => /lib/x86_64-linux-gnu/libgpg-error.so.0 (0x00007f899eb60000)
	libtirpc.so.3 => /lib/x86_64-linux-gnu/libtirpc.so.3 (0x00007f899eb32000)
	libgssapi_krb5.so.2 => /lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007f899eade000)
	libkrb5.so.3 => /lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007f899ea13000)
	libk5crypto.so.3 => /lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007f899e9e4000)
	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007f899e9dc000)
	libkrb5support.so.0 => /lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007f899e9ce000)
	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007f899e9c7000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f899e9b3000)
```

Reply:
I see how to do  this now - thanks!

```
$ dpkg -I libsystemd0_249.11-0ubuntu3.9_amd64.deb 
 new Debian package, version 2.0.
 size 317540 bytes: control archive=4288 bytes.
     652 bytes,    14 lines      control              
     230 bytes,     3 lines      md5sums              
      37 bytes,     1 lines      shlibs               
   27477 bytes,   627 lines      symbols              
      72 bytes,     2 lines      triggers             
 Package: libsystemd0
 Source: systemd
 Version: 249.11-0ubuntu3.9
 Architecture: amd64
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Installed-Size: 995
 Pre-Depends: libc6 (>= 2.34), libcap2 (>= 1:2.24-9~), libgcrypt20 (>= 1.9.0), liblz4-1 (>= 0.0~r122), liblzma5 (>= 5.1.1alpha+20120614), libzstd1 (>= 1.4.0)
 Section: libs
...
```

I will use this method from now on.